### PR TITLE
feat: changes to page templates

### PR DESCRIPTION
**Pull request by sourcefilter**
connected to
https://jira.library.ucla.edu/browse/APPS-1802
https://jira.library.ucla.edu/browse/APPS-660
https://jira.library.ucla.edu/browse/APPS-2287
https://jira.library.ucla.edu/browse/APPS-1801

### CODE REVIEW

File: app/views/static/manuscript_descriptions.html.erb [hunk 1]
```diff
@@ -20,32 +20,51 @@
...
+        </p>
+
+        <p class='static-page__text static-page__text--sinai'>
+          When performing a “Title” search on the Sinai Manuscripts Digital Library website, both the “Works” field and the “Contents note” fields are searched.
```
Comment (kirschbombe) [inline app/views/static/manuscript_descriptions.html.erb:60]: Sorry - I forgot to edit this bc we just introduced a new field. This line should be: When performing a "Title" search on the Sinai Manuscripts Digital Library website, the "Works', "Contents", and "Contents note" fields are searched.

File: app/views/static/references.html.erb [hunk 2]
```diff
@@ -9,30 +9,81 @@
       </div>
 
       <div class='static-page__row'>
+        <h2 class='static-page__subtitle static-page__subtitle--sinai'>Catalogues and Handlists</h2>
 
-        <p class='static-page__text static-page__text--sinai'>The following catalogues and handlists were consulted in creating the descriptive metadata for the Syriac and Arabic manuscripts published through the Sinai Manuscripts Digital Library website.</p>
+        <p class="static-page__text static-page__text--sinai">The following catalogues and handlistswere consulted in creating the descriptive metadata for the manuscripts published through the Sinai Manuscripts Digital Library website.</p>
```
Comment (wlpotter) [inline app/views/static/references.html.erb:14]: space needed between "handlistswere" -> "handlists were"

File: app/views/static/references.html.erb [hunk 3]
```diff
@@ -9,30 +9,81 @@
...
 
-        <p class='static-page__text static-page__text--sinai'>Atiya = Atiya, Aziz Suryal. <i>The Arabic Manuscripts of Mount Sinai: A Hand-List of the Arabic Manuscripts and Scrolls Microfilmed at the Library of the Monastery of St. Catherine, Mount Sinai.</i> Johns Hopkins Press, 1955. [<%= link_to 'Internet Archive', "https://archive.org/details/arabicmanuscript0000atiy_m0l2/mode/2up", class: 'si-link' %>]</p>
+        <p class='static-page__text static-page__text--sinai'>Aland = Aland, K. <i>Kurzgefasste Liste der griechischen Handschriften des Neuen Testaments</i>. Berlin: De Gruyter, 1963, rev. ed. 1994. [<%= link_to 'WorldCat', 'https://www.worldcat.org/oclc/32035214&amp;sa=D&amp;source=editors&amp;ust=1683922505975654&amp;usg=AOvVaw2RrAa4ir7so9m1wEDFxrFp"', class: 'si-link' %>]</p>
```
Comment (wlpotter) [inline app/views/static/references.html.erb:16]: I noticed this throughout, but the URLs in the links have a long string of parameters add at the end e.g. `&amp;sa=D&amp;source=editors&amp;ust=1683922505975654&amp;usg=AOvVaw2RrAa4ir7so9m1wEDFxrFp"`. In most cases this is causing the links not to resolve

Comment (wlpotter) [reply app/views/static/references.html.erb:16]: I believe all of the links on this page have been affected

File: app/views/static/references.html.erb [hunk 4]
```diff
@@ -9,30 +9,81 @@
...
 
-        <p class='static-page__text static-page__text--sinai'>Géhin 2017 = Géhin, Paul. <i>Les manuscrits syriaques de parchemin du Sinaï et leurs</i> membra disjecta (CSCO 665, Subsidia 136; Leuven, 2017). [<%= link_to 'WorldCat', "https://www.worldcat.org/title/manuscrits-syriaques-de-parchemin-du-sinai-et-leurs-membra-disjecta/oclc/1000455293&referer=brief_results", class: 'si-link' %>]</p>
+        <p class='static-page__text static-page__text--sinai'>Brock 1995 = S.P. Brock. <i>Catalogue of Syriac Fragments (New Finds) in the Library of the Monastery of St Catherine, Mount Sinai</i>(Athens, 1995). [<%= link_to 'WorldCat', 'https://www.worldcat.org/title/catalogue-of-syriac-fragments-new-finds-in-the-library-of-the-monastery-of-saint-catherine-mount-sinai/oclc/780492566?referer%3Ddi%26ht%3Dedition&amp;sa=D&amp;source=editors&amp;ust=1683922505977880&amp;usg=AOvVaw3YjFUlR-yRhXfx2L9dpFxz"', class: 'si-link' %>]</p>
```
Comment (wlpotter) [inline app/views/static/references.html.erb:24]: Missing space: `Sinai</i>(Athens` -> `Sinai</i> (Athens`

File: app/views/static/references.html.erb [hunk 5]
```diff
@@ -9,30 +9,81 @@
...
 
-        <p class='static-page__text static-page__text--sinai'>Kamil = Kāmil, Murād. <i>Catalogue of All Manuscripts in the Monastery of St. Catherine on Mount Sinai.</i> Harrassowitz, 1970. [<%= link_to 'WorldCat', "https://www.worldcat.org/title/catalogue-of-all-manuscripts-in-the-monastery-of-st-catharine-on-mount-sinai/oclc/859613867?referer=br&ht=edition", class: 'si-link' %>]</p>
+        <p class='static-page__text static-page__text--sinai'>G&eacute;hin 2006= G&eacute;hin, Paul. &ldquo;Manuscrits sina&iuml;tiques dispers&eacute;s, I: les fragments arabes et syriaques de Paris.&rdquo; <i>OC</i>90 (2006): 23-43. [<%= link_to 'WorldCat', 'https://www.worldcat.org/title/-manuscrits-sinaitiques-disperses-i-les-fragments-syriaques-et-arabes-de-paris/oclc/784157423%26referer%3Dbrief_results&amp;sa=D&amp;source=editors&amp;ust=1683922505979436&amp;usg=AOvVaw1uYIf9GxuZi6FqmEf8cZfR"', class: 'si-link' %>]</p>
```
Comment (wlpotter) [inline app/views/static/references.html.erb:30]: Missing space: `G&eacute;hin 2006= G&eacute;hin` -> `G&eacute;hin 2006 = G&eacute;hin`

Missing space: `<i>OC</i>90` -> `<i>OC</i> 90`

File: app/views/static/references.html.erb [hunk 6]
```diff
@@ -9,30 +9,81 @@
...
 
-        <p class='static-page__text static-page__text--sinai'>Lewis = Lewis, Agnes Smith. <i>Catalogue of the Syriac Manuscripts in the Convent of Saint Catherine on Mount Sinai.</i> Cambridge University Press, 1894. [<%= link_to 'Internet Archive', "https://archive.org/details/catalogueofsyria00sainrich", class: 'si-link' %>]</p>
+        <p class='static-page__text static-page__text--sinai'>G&eacute;hin 2009 = G&eacute;hin, Paul. &quot;Fragments patristiques syriaques des Nouvelles d&eacute;couvertes du Sina&iuml;.&quot; <i>Collectanea Christiana Orientalia</i>6 (2009): 67-93</p>
```
Comment (wlpotter) [inline app/views/static/references.html.erb:32]: Missing space: `Orientalia</i>6` -> `Orientalia</i> 6`

File: app/views/static/references.html.erb [hunk 7]
```diff
@@ -9,30 +9,81 @@
...
 
-        <p class='static-page__text static-page__text--sinai'>Meimaris = Meimaris, Yiannis E. <i>Katalogos ton neon aravikon cheirographon tes Hieras Mones Hagias Aikaterines tou Orous Sina = Kataluy al-Majtutat al-’arabiyya al-muktasafa haditan bi-Dayr Sant Katarin al-muqaddas.</i> Ethnikon Hidryma Ereunon, 1985. [<%= link_to 'WorldCat', "https://www.worldcat.org/title/katalogos-ton-neon-aravikon-cheirographon-tes-hieras-mones-hagias-aikaterines-tou-orous-sina-kataluy-al-majtutat-al-arabiyya-al-muktasafa-haditan-bi-dayr-sant-katarin-al-muqaddas/oclc/1123922473?referer=br&ht=edition", class: 'si-link' %>]</p>
+        <p class='static-page__text static-page__text--sinai'>G&eacute;hin 2017 = G&eacute;hin, Paul. <i>Les manuscrits syriaques de parchemin du Sina&iuml; et leurs</i>membra disjecta (CSCO 665, Subsidia 136; Leuven, 2017). [<%= link_to 'WorldCat', 'https://www.worldcat.org/title/manuscrits-syriaques-de-parchemin-du-sinai-et-leurs-membra-disjecta/oclc/1000455293%26referer%3Dbrief_results&amp;sa=D&amp;source=editors&amp;ust=1683922505980598&amp;usg=AOvVaw2ZbqofVswP2V57Y1V5Z1Z_"', class: 'si-link' %>]</p>
```
Comment (wlpotter) [inline app/views/static/references.html.erb:34]: Missing space: `et leurs</i>membra` -> `et leurs</i> membra`

File: app/views/static/references.html.erb [hunk 8]
```diff
@@ -9,30 +9,81 @@
...
+
+        <p class='static-page__text static-page__text--sinai'>Kamil = K&#257;mil, Mur&#257;d. <i>Catalogue of All Manuscripts in the Monastery of St. Catherine on Mount Sinai</i>. Harrassowitz, 1970. [<%= link_to 'WorldCat', 'https://www.worldcat.org/title/catalogue-of-all-manuscripts-in-the-monastery-of-st-catharine-on-mount-sinai/oclc/859613867?referer%3Dbr%26ht%3Dedition&amp;sa=D&amp;source=editors&amp;ust=1683922505982771&amp;usg=AOvVaw2eubcGMhjKIcgPk9rRia8C"', class: 'si-link' %>] [<%= link_to 'WorldCat', 'https://www.worldcat.org/oclc/42432167&amp;sa=D&amp;source=editors&amp;ust=1683922505983084&amp;usg=AOvVaw38nfNYq_H1cjtKjVMq3sbv"', class: 'si-link' %>]</p>
```
Comment (wlpotter) [inline app/views/static/references.html.erb:40]: The second link here, `[<%= link_to 'WorldCat', 'https://www.worldcat.org/oclc/42432167&amp;sa=D&amp;source=editors&amp;ust=1683922505983084&amp;usg=AOvVaw38nfNYq_H1cjtKjVMq3sbv"', class: 'si-link' %>]` can be removed. This was my fault as it slipped through my proofreading of the Google doc, apologies

File: app/views/static/references.html.erb [hunk 9]
```diff
@@ -9,30 +9,81 @@
...
+
+        <p class='static-page__text static-page__text--sinai'>Lewis = Lewis, Agnes Smith. <i>Catalogue of the Syriac Manuscripts in the Convent of Saint Catherine on Mount Sinai</i>. Cambridge University Press, 1894. [<%= link_to 'Internet Archive', 'https://archive.org/details/catalogueofsyria00sainrich&amp;sa=D&amp;source=editors&amp;ust=1683922505983791&amp;usg=AOvVaw1Q37h4ADZ3K3zCnnsEG7Ij"', class: 'si-link' %>]</p>
+
+        <p class='static-page__text static-page__text--sinai'>LOC= <i>Manuscripts in St. Catherine&#39;s Monastery, Mount Sinai</i>. Library of Congress. [<%= link_to 'Library of Congress', 'https://www.loc.gov/collections/manuscripts-in-st-catherines-monastery-mount-sinai&amp;sa=D&amp;source=editors&amp;ust=1683922505984517&amp;usg=AOvVaw19FeW8w9OzVFkzT3mhQFdm"', class: 'si-link' %>]</p>
```
Comment (wlpotter) [inline app/views/static/references.html.erb:44]: Missing space: `LOC= ` -> `LOC = `

File: app/views/static/references.html.erb [hunk 10]
```diff
@@ -9,30 +9,81 @@
...
+      </div>
+      <hr class='divider'>
+      <div class='static-page__row'>
+        <h2 class='static-page__subtitle static-page__subtitle--sinai'>Additional Sources</h2>
+
+        <p class='static-page__text static-page__text--sinai'>The following sources were also consulted in creating the descriptive metadata for the manuscripts published through the Sinai Manuscripts Digital Library website.</p>
+
+        <p class='static-page__text static-page__text--sinai'>Binggeli = Binggeli, Andr&eacute;. &quot;Les trois David copistes arabes de Palestine aux 9e-10e s.&rdquo; In <i>Manuscripta graeca et orientalia: M&eacute;langes monastiques et patristiques en l&rsquo;honneur de Paul G&eacute;hin</i>. Edited by Andr&eacute; Binggeli, Anne Boud&#39;hors, and Matthieu Cassin, 79-118. Leuven: Peeters, 2016.</p>
+
+        <p class='static-page__text static-page__text--sinai'>Brock 2003 = Brock, S. &ldquo;Syriac on Sinai: The Main Connections.&rdquo; In <i>Eukosmia: Studi miscellanei per il 75&deg; di Vincenzo Poggi S.J</i>. Edited by V. Ruggieri and L. Pieralli, 103-117. Catanzaro, 2003).</p>
```
Comment (wlpotter) [inline app/views/static/references.html.erb:62]: Typo (again on my end, apologies): `Catanzaro, 2003).` -> `Catanzaro, 2003.`

File: app/views/static/references.html.erb [hunk 11]
```diff
@@ -9,30 +9,81 @@
...
+
+        <p class='static-page__text static-page__text--sinai'>Brock 2011 = Brock, Sebastian P. &quot;The Syriac &#39;New Finds&#39; at St Catherine&#39;s Monastery, Sinai, and Their Significance.&quot; <i>The Harp</i>27 (2011): 39-52.</p>
+
+        <p class='static-page__text static-page__text--sinai'>Brock 2012 = Brock, S. P. &ldquo;Sinai: A Meeting Point of Georgian with Syriac and Christian Palestinian Aramaic.&rdquo; In <i>Caucasus: Between East and West. Historical and Philological Studies in Honour of Zaza Aleksidze</i>, 482-494. Tbilisi: National Centre of Manuscripts, 2012.</p>
+
+        <p class='static-page__text static-page__text--sinai'>Froyshov = Froyshov, S. R. &ldquo;L&rsquo;horloge &#39;g&eacute;orgien&#39; du Sinaiticus ibericus 34.&rdquo; PhD diss., Paris IV-Sorbonne, 2003.</p>
+
+        <p class='static-page__text static-page__text--sinai'>Kashouh 2008 = Kashouh, Hikmat. &ldquo;Sinai Ar. N.F. Parchment 8 and 28: Its Contribution to Textual Criticism of the Gospel of Luke.&rdquo; <i>Novum Testamentum</i>50 (2008): 28-57.</p>
```
Comment (wlpotter) [inline app/views/static/references.html.erb:70]: Missing space: `<i>Novum Testamentum</i>50` -> `<i>Novum Testamentum</i> 50`

File: app/views/static/references.html.erb (hunk 3 above)
Comment (wlpotter) [reply app/views/static/references.html.erb:16]: In this case it should be `https://www.worldcat.org/oclc/32035214`. I can go through and include the correct links for each

File: app/views/static/references.html.erb [hunk 12]
```diff
@@ -9,30 +9,81 @@
...
 
-        <p class='static-page__text static-page__text--sinai'>Brock, Cat. = S.P. Brock. <i>Catalogue of Syriac Fragments (New Finds) in the Library of the Monastery of St Catherine, Mount Sinai</i> (Athens, 1995). [<%= link_to 'WorldCat', "https://www.worldcat.org/title/catalogue-of-syriac-fragments-new-finds-in-the-library-of-the-monastery-of-saint-catherine-mount-sinai/oclc/780492566?referer=di&ht=edition", class: 'si-link' %>]</p>
+        <p class='static-page__text static-page__text--sinai'>Aleksidze, et al. = Aleksidze, Z., et al. <i>Catalogue of Georgian manuscripts discovered in 1975 at St. Catherine&#39;s Monastery on Mount Sinai</i>. Athens, 2005. [<%= link_to 'WorldCat', 'https://www.worldcat.org/oclc/1131665083&amp;sa=D&amp;source=editors&amp;ust=1683922505976172&amp;usg=AOvVaw3i-kfj6WhY44OytwlAWjP8"', class: 'si-link' %>]</p>
```
Comment (wlpotter) [inline app/views/static/references.html.erb:18]: URL should be https://www.worldcat.org/oclc/1131665083

File: app/views/static/references.html.erb (hunk 4 above)
Comment (wlpotter) [reply app/views/static/references.html.erb:24]: URL should be: https://www.worldcat.org/title/catalogue-of-syriac-fragments-new-finds-in-the-library-of-the-monastery-of-saint-catherine-mount-sinai/oclc/780492566

File: app/views/static/references.html.erb [hunk 13]
```diff
@@ -9,30 +9,81 @@
...
 
-        <p class='static-page__text static-page__text--sinai'>Gibson = Gibson, Margaret Dunlop. <i>Catalogue of the Arabic Manuscripts in the Convent of S. Catharine on Mount Sinai.</i> Cambridge University Press, 1894. [<%= link_to 'Internet Archive', "https://archive.org/details/catalogueofarabi00sainrich", class: 'si-link' %>]</p>
+        <p class='static-page__text static-page__text--sinai'>Clark =Clark, Kenneth W., ed. <i>Checklist of Manuscripts in St. Catherine&rsquo;s Monastery Mount Sinai. </i>Washington, D.C.: Library of Congress, 1952. [<%= link_to 'Library of Congress', 'https://www.loc.gov/item/52060038&amp;sa=D&amp;source=editors&amp;ust=1683922505978433&amp;usg=AOvVaw11rSYTSzGBp4-g0GroprM9"', class: 'si-link' %>]</p>
```
[the remark this answers is not in the document]
Comment (wlpotter) [reply app/views/static/references.html.erb:26]: URL should be: https://www.loc.gov/item/52060038

File: app/views/static/references.html.erb [hunk 14]
```diff
@@ -9,30 +9,81 @@
...
 
-        <p class='static-page__text static-page__text--sinai'>Greek New Finds = Nikolopoulos, P. G. Τα νέα ευρήματα του Σινά (Athens, 1998). English translation: A. Hatzopoulos, The New Finds of Sinai (Athens, 1999). [<%= link_to 'WorldCat', "https://www.worldcat.org/title/new-finds-of-sinai/oclc/913489510&referer=brief_results", class: 'si-link' %>]</p>
+        <p class='static-page__text static-page__text--sinai'>Gardthausen = Gardthausen, V. E. <i>Catalogus codicum Graecorum Sinaiticorum</i>. Oxford, 1886. [<%= link_to 'Internet Archive', 'https://archive.org/details/cataloguscodicu00sinagoog&amp;sa=D&amp;source=editors&amp;ust=1683922505978724&amp;usg=AOvVaw3mvUuQiIh3yasFgHnY7jTX"', class: 'si-link' %>]</p>
```
Comment (wlpotter) [inline app/views/static/references.html.erb:28]: URL should be: https://archive.org/details/cataloguscodicu00sinagoog

File: app/views/static/references.html.erb (hunk 5 above)
Comment (wlpotter) [reply app/views/static/references.html.erb:30]: URL should be: https://www.worldcat.org/title/-manuscrits-sinaitiques-disperses-i-les-fragments-syriaques-et-arabes-de-paris/oclc/784157423

File: app/views/static/references.html.erb (hunk 7 above)
Comment (wlpotter) [reply app/views/static/references.html.erb:34]: URL should be: https://www.worldcat.org/title/manuscrits-syriaques-de-parchemin-du-sinai-et-leurs-membra-disjecta/oclc/1000455293

File: app/views/static/references.html.erb [hunk 15]
```diff
@@ -9,30 +9,81 @@
...
 
-        <p class='static-page__text static-page__text--sinai'>Philothée 2008 = Philothée du Sinaï, <i>Nouveaux manuscrits syriaques du Sinaï</i> (Athens, 2008). [<%= link_to 'WorldCat', "https://www.worldcat.org/title/nouveaux-manuscrits-syriaques-du-sinai/oclc/476756980?referer=br&ht=edition", class: 'si-link' %>]</p>
+        <p class='static-page__text static-page__text--sinai'>Gibson = Gibson, Margaret Dunlop. <i>Catalogue of the Arabic Manuscripts in the Convent of S. Catharine on Mount Sinai</i>. Cambridge University Press, 1894.[<%= link_to 'Internet Archive', 'https://archive.org/details/catalogueofarabi00sainrich&amp;sa=D&amp;source=editors&amp;ust=1683922505981437&amp;usg=AOvVaw0O-pBD3AlmXKj057ALKt5h"', class: 'si-link' %>]</p>
```
Comment (wlpotter) [inline app/views/static/references.html.erb:36]: URL should be: https://archive.org/details/catalogueofarabi00sainrich

File: app/views/static/references.html.erb (hunk 8 above)
Comment (wlpotter) [reply app/views/static/references.html.erb:40]: The URL for the first link should be: https://www.worldcat.org/title/catalogue-of-all-manuscripts-in-the-monastery-of-st-catharine-on-mount-sinai/oclc/859613867

File: app/views/static/references.html.erb (hunk 9 above)
Comment (wlpotter) [reply app/views/static/references.html.erb:44]: URL should be: https://www.loc.gov/collections/manuscripts-in-st-catherines-monastery-mount-sinai

File: app/views/static/references.html.erb [hunk 16]
```diff
@@ -9,30 +9,81 @@
...
+
+        <p class='static-page__text static-page__text--sinai'>LOC= <i>Manuscripts in St. Catherine&#39;s Monastery, Mount Sinai</i>. Library of Congress. [<%= link_to 'Library of Congress', 'https://www.loc.gov/collections/manuscripts-in-st-catherines-monastery-mount-sinai&amp;sa=D&amp;source=editors&amp;ust=1683922505984517&amp;usg=AOvVaw19FeW8w9OzVFkzT3mhQFdm"', class: 'si-link' %>]</p>
+
+        <p class='static-page__text static-page__text--sinai'>Meimaris = Meimaris, Yiannis E. <i>Katalogos ton neon aravikon cheirographon tes Hieras Mones Hagias Aikaterines tou Orous Sina = Kataluy al-Majtutat al-&rsquo;arabiyya al-muktasafa haditan bi-Dayr Sant Katarin al-muqaddas</i>. Ethnikon Hidryma Ereunon, 1985. [<%= link_to 'WorldCat', 'https://www.worldcat.org/title/katalogos-ton-neon-aravikon-cheirographon-tes-hieras-mones-hagias-aikaterines-tou-orous-sina-kataluy-al-majtutat-al-arabiyya-al-muktasafa-haditan-bi-dayr-sant-katarin-al-muqaddas/oclc/1123922473?referer%3Dbr%26ht%3Dedition&amp;sa=D&amp;source=editors&amp;ust=1683922505985361&amp;usg=AOvVaw0xshNif-zj3YjExAa21Nsr"', class: 'si-link' %>]</p>
```
Comment (wlpotter) [inline app/views/static/references.html.erb:46]: URL should be: https://www.worldcat.org/title/katalogos-ton-neon-aravikon-cheirographon-tes-hieras-mones-hagias-aikaterines-tou-orous-sina-kataluy-al-majtutat-al-arabiyya-al-muktasafa-haditan-bi-dayr-sant-katarin-al-muqaddas/oclc/1123922473

File: app/views/static/references.html.erb [hunk 17]
```diff
@@ -9,30 +9,81 @@
...
+
+        <p class='static-page__text static-page__text--sinai'>NLI = Manuscripts in the Holy Monastery of St. Catherine at Mount Sinai. National Library of Israel.</p>
+
+        <p class='static-page__text static-page__text--sinai'>Perria = Perria, Lidia. <i>Repertorio dei manoscritti greci di area orientale, palestino-sinaitica</i>. Messina, 2000. [<%= link_to 'WorldCat', 'https://www.worldcat.org/oclc/1158454955&amp;sa=D&amp;source=editors&amp;ust=1683922505985952&amp;usg=AOvVaw3NtqtWZZx_LGdhjMeFPVwj"', class: 'si-link' %>]</p>
```
Comment (wlpotter) [inline app/views/static/references.html.erb:50]: URL should be: https://www.worldcat.org/oclc/1158454955

File: app/views/static/references.html.erb [hunk 18]
```diff
@@ -9,30 +9,81 @@
...
+
+        <p class='static-page__text static-page__text--sinai'>Froyshov = Froyshov, S. R. &ldquo;L&rsquo;horloge &#39;g&eacute;orgien&#39; du Sinaiticus ibericus 34.&rdquo; PhD diss., Paris IV-Sorbonne, 2003.</p>
+
+        <p class='static-page__text static-page__text--sinai'>Kashouh 2008 = Kashouh, Hikmat. &ldquo;Sinai Ar. N.F. Parchment 8 and 28: Its Contribution to Textual Criticism of the Gospel of Luke.&rdquo; <i>Novum Testamentum</i>50 (2008): 28-57.</p>
+
+        <p class='static-page__text static-page__text--sinai'>Kashouh 2012 = Kashouh, Hikmat. <i>The Arabic Versions of the Gospels: The Manuscripts and their Families</i>. Berlin: De Gruyter, 2012. [<%= link_to 'WorldCat', 'https://www.worldcat.org/title/671466151&amp;sa=D&amp;source=editors&amp;ust=1683922505987959&amp;usg=AOvVaw3ZNYDFPFMuB3Bb0fqCVBpJ"', class: 'si-link' %>]</p>
```
Comment (wlpotter) [inline app/views/static/references.html.erb:72]: URL should be: https://www.worldcat.org/title/671466151

File: app/views/static/references.html.erb [hunk 19]
```diff
@@ -9,30 +9,81 @@
...
+
+        <p class='static-page__text static-page__text--sinai'>Lamoreaux = Lamoreaux, J.C., ed. <i>The Life of Stephen of Mar Sabas</i>. Leuven: Peeters, 1999. [<%= link_to 'WorldCat', 'https://www.worldcat.org/oclc/42432167&amp;sa=D&amp;source=editors&amp;ust=1683922505988301&amp;usg=AOvVaw2GHdkqfAbY6QerwaKpFZ1n"', class: 'si-link' %>]</p>
```
Comment (wlpotter) [inline app/views/static/references.html.erb:74]: URL should be: https://www.worldcat.org/oclc/42432167

File: app/views/static/references.html.erb [hunk 20]
```diff
@@ -9,30 +9,81 @@
...
+
+        <p class='static-page__text static-page__text--sinai'>Metreveli = Metreveli, E. <i>Dzlisp&#39;irni da &#289;mrtism&scaron;oblisani</i>. Tbilisi, 1971. [<%= link_to 'WorldCat', 'https://www.worldcat.org/oclc/68159930&amp;sa=D&amp;source=editors&amp;ust=1683922505988646&amp;usg=AOvVaw1ays6uBrWE7El16JPiK-AE"', class: 'si-link' %>]</p>
```
Comment (wlpotter) [inline app/views/static/references.html.erb:76]: URL should be: https://www.worldcat.org/oclc/68159930

File: app/views/static/references.html.erb [hunk 21]
```diff
@@ -9,30 +9,81 @@
...
+
+        <p class='static-page__text static-page__text--sinai'>Miklas 2000 = Miklas, H. &ldquo;Zur editorischen Vorbereitung des sog. Missale Sinaiticum (Sin. slav. 5/N).&rdquo; In <i>Glagolitica: Zum Ursprung der slavischen Schriftkultur</i>. Edited by H. Miklas, Sylvia Richter, and Veliza Sadovski, 117-129. Wien: Verlag der &Ouml;sterreichischen Akademie der Wissenschaften, 2000.</p>
+
+        <p class='static-page__text static-page__text--sinai'>Miklas, et al. = Miklas, H., et al. <i>Psalterium Demetrii Sinaitici (monasterii sanctae Catharinae codex slav. 3/N)</i>. Glagolitica Sinaitica 1. Vienna, 2012. [<%= link_to 'WorldCat', 'https://www.worldcat.org/oclc/824728170&amp;sa=D&amp;source=editors&amp;ust=1683922505989191&amp;usg=AOvVaw1MSubrJ4A8Qbeu1lwcTxsI"', class: 'si-link' %>]</p>
```
Comment (wlpotter) [inline app/views/static/references.html.erb:80]: URL should be: https://www.worldcat.org/oclc/824728170

File: app/views/static/references.html.erb [hunk 22]
```diff
@@ -9,30 +9,81 @@
...
+
+        <p class='static-page__text static-page__text--sinai'>Rossetto = Rossetto, G. <i>Greek Palimpsests at Saint Catherine&rsquo;s Monastery (Sinai): Three Euchologia as Case Studies</i>. Ver&ouml;ffentlichungen zur Byzanzforschung 44. Wien: Austrian Academy of Sciences Press, 2023. [<%= link_to 'WorldCat', 'https://www.worldcat.org/oclc/1370165262&amp;sa=D&amp;source=editors&amp;ust=1683922505989599&amp;usg=AOvVaw1lGrYiu7YoZ21XInE0jZ_P"', class: 'si-link' %>]</p>
```
Comment (wlpotter) [inline app/views/static/references.html.erb:82]: URL should be: https://www.worldcat.org/oclc/1370165262

File: app/views/static/references.html.erb [hunk 23]
```diff
@@ -9,30 +9,81 @@
...
+
+        <p class='static-page__text static-page__text--sinai'>SIBE = Gippert, J., W. Schulze, Z. Aleksidze, and J.-P. Mah&eacute;, eds. <i>The Caucasian Albanian Palimpsests of Mount Sinai</i>. Monumenta Palaeographica Medii Aevi: Series Ibero-Caucasica 2. Turnhout, 2008. [<%= link_to 'WorldCat', 'https://www.worldcat.org/oclc/319126785&amp;sa=D&amp;source=editors&amp;ust=1683922505989941&amp;usg=AOvVaw3RSeARVWuhm-zPpQbjN603"', class: 'si-link' %>]</p>
```
Comment (wlpotter) [inline app/views/static/references.html.erb:84]: URL should be: https://www.worldcat.org/oclc/319126785

**Review by pghorpade — APPROVED** [review]
👍

**Review by pghorpade — APPROVED** [review]
👍

**Pull request merged**